tests/todo/T2076.hs

```diff
@@ -0,0 +1,17 @@
+module T2076 where 
```

Comment:
Should this be placed outside of `tests/todo` if it passes?

Also, this needs to be listed in `tests/tests.cabal` for CI to pick it up.